tests/test_ell1h.py

```diff
@@ -216,3 +228,25 @@ def test_zero_H3_H4_fit_H3(toasJ0613):
     # This should work
     f.fit_toas()
     assert f.model.H3.value > 0.0
+
+
+@pytest.mark.parametrize("model", ["modelJ1853", "modelJ0613"])
+def test_stand_alone_model_params(model, request):
```

Comment:
What property are you actually checking here? Looks like maybe the same `verify_` something function would cover most of this?